src/react-apps/applications/ux-editor/src/containers/Container.tsx

```diff
@@ -178,14 +179,9 @@ export class ContainerComponent extends React.Component<IContainerProps> {
   }
 
   public handleAddNewGroup = () => {
-    const container: ICreateFormContainer = {
-      repeating: this.props.repeating,
-      dataModelGroup: this.props.dataModelGroup,
-      index: this.props.index + 1,
-    };
-
-    FormDesignerActionDispatchers.addFormContainer(container, this.props.id);
+    FormDesignerActionDispatchers.createRepeatingGroup(this.props.id);
```

Comment:
What about containers that are not repeating?

Reply:
The handleAddNewGroup() method is only referenced from the renderNewGroupButton(), which is only rendered for repeating groups.